contracts/sol6/fprConversionRates/QtyStepConversionRates.sol

```diff
@@ -0,0 +1,436 @@
+pragma solidity 0.6.6;
+
+import "../IERC20.sol";
```

Comment:
redundant since imported in conversion rates

Reply:
removed